schema/tables.sql

```diff
@@ -1885,7 +1947,7 @@ CREATE TABLE `Dewar` (
   `transportValue` int(11) unsigned DEFAULT NULL,
   `trackingNumberToSynchrotron` varchar(30) DEFAULT NULL,
   `trackingNumberFromSynchrotron` varchar(30) DEFAULT NULL,
-  `type` enum('Dewar','Toolbox') NOT NULL DEFAULT 'Dewar',
+  `type` enum('Dewar','Toolbox','Parcel') NOT NULL DEFAULT 'Dewar',
```

Comment:
Need to extend the shipping interface to allow selection of dewar type, this is completely abstracted at the moment